src/ansys/fluent/core/session_base_meshing.py

```diff
@@ -146,10 +147,20 @@ def workflow(self):
         return self._old_workflow
 
     @property
-    def new_workflow(self):
+    def new_wt_workflow(self):
         """Datamodel root of workflow exposed in object-oriented manner."""
         if not self._new_workflow:
-            self._new_workflow = EnhancedMeshingWorkflow(
+            self._new_workflow = WaterTightMeshingWorkflow(
+                self._workflow_se,
+                self.meshing,
+            )
+        return self._new_workflow
+
+    @property
+    def new_ft_workflow(self):
```

Comment:
`fault_tolerant_workflow`

Reply:
If it's always the same cached object, how can you truly switch workflows? Even if you maintain two separate objects for wt and ft respectively, you still need to be careful to avoid a caching problem when you try to revert back to a given workflow type because the cached object would be invalid.